lighthouse-core/audits/time-to-interactive.js

```diff
@@ -63,9 +63,6 @@ class TTIMetric extends Audit {
         return {currentLatency, foundLatencies};
       }
 
-      // Clip to the trace end
-      endTime = Math.min(endTime, maxTime);
```

Comment:
hmm! why'd you nuke this?
 
fwiw: https://docs.google.com/document/d/1b9slyaB9yho91YTOkAQfpCdULFkZM9LqsipcX3t7He8/edit#heading=h.ooalxriljq1i has details on the clipping approach for EIL

cc @brendankenny